### PR TITLE
i18n(es): add render

### PR DESCRIPTION
**Pull request by kevinzunigacuellar**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description

- add `render.md` in spanish

### CODE REVIEW

File: src/pages/es/guides/deploy/render.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: Despliega tu projecto de Astro en render
```
Comment (fredoist) [inline src/pages/es/guides/deploy/render.md:2]:
```suggestion
title: Despliega tu proyecto de Astro en Render
```

File: src/pages/es/guides/deploy/render.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+description: Cómo desplegar tu projecto de Astro en render
```
Comment (fredoist) [inline src/pages/es/guides/deploy/render.md:3]:
```suggestion
description: Cómo desplegar tu proyecto de Astro en Render
```

**Review by fredoist — CHANGES_REQUESTED** [review]
Algunas correcciones menores.

File: src/pages/es/guides/deploy/render.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+description: Cómo desplegar tu proyecto de Astro en Render
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+Puedes desplegar tu proyecto de Astro en [Render](https://render.com/), un servicio para crear sitios web con certificados TLS gratuitos, una CDN global, protección DDoS, redes privadas y despliegues automáticos desde Git.
+
+## Cómo desplegar
+
+1. Crea una [cuenta de render.com](https://dashboard.render.com/) e inicie sesión
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/render.md:12]:
```suggestion
1. Crea una [cuenta de render.com](https://dashboard.render.com/) e inicia sesión
```

File: src/pages/es/guides/deploy/render.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+2. Haga clic en el botón **New +** en el dashboard y seleccione **Static Site**
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/render.md:13]:
```suggestion
2. Haz clic en el botón **New +** en el dashboard y selecciona **Static Site**
```

File: src/pages/es/guides/deploy/render.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+3. Conecta tu repositorio de [GitHub](https://github.com/) o [GitLab](https://about.gitlab.com/) o, alternativamente, ingrese la URL pública del repositorio
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/render.md:14]:
```suggestion
3. Conecta tu repositorio de [GitHub](https://github.com/) o [GitLab](https://about.gitlab.com/) o, alternativamente, ingresa la URL pública del repositorio
```

File: src/pages/es/guides/deploy/render.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+4. Asigne un nombre a tu sitio web, seleccione la rama y especifique el comando de compilación y el directorio de publicación
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/render.md:15]:
```suggestion
4. Asigna un nombre a tu sitio web, selecciona la rama y especifica el comando de compilación y el directorio de publicación
```

File: src/pages/es/guides/deploy/render.md [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+    - **build command:** `npm run build`
+    - **publish directory:** `dist`
+5. Haga clic en el botón **Create Static Site**
```
Comment (agustinmulet) [inline src/pages/es/guides/deploy/render.md:18]:
```suggestion
    - **publish directory:** `dist`
5. Haz clic en el botón **Create Static Site**

```

**Review by agustinmulet — COMMENTED** [review]
Revisado! Buen trabajo 💪

**Comment by kevinzunigacuellar** [comment]
good to merge 😇

**Comment by Yan-Thomas** [comment]
Good job squad! Thanks for this 💜

**Pull request merged**